db/primary_migrate/20221109165826_create_irs_attempt_api_log_files.rb

```diff
@@ -0,0 +1,12 @@
+class CreateIrsAttemptApiLogFiles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :irs_attempt_api_log_files do |t|
+      t.string :filename
+      t.string :iv
+      t.text :encrypted_key
+      t.datetime :requested_time
```

Comment:
if we're going to do a Db table, should we add an index on requested time? what is the thing we're going to look these up by?

Also would it make sense to make this a string? Because we're going to do exact lookups, and doing equality on datetimes can be challenging? Or maybe drop down the `precision` in the DB (such as to whole seconds, or something)?